lib/openhab/dsl/rules/builder.rb

```diff
@@ -548,6 +551,107 @@ def dependencies(trigger_types = %i[changed updated])
           raise ArgumentError, "Object passed to not_if must be a proc" unless item.is_a?(Proc)
         end
 
+        #
+        # Prevents or delays executions of rules to within a specified interval.
+        # Debounce handling is done after the from/to/command types are filtered, but before only_if/not_if
```

Comment:
Need a newline between these two sentences so that YARD only includes the first sentence as the "short" summary for this method.

Reply:
Actually, YARD seems to extract the first sentence regardless. In any case, I've made this private in the next iteration.